packages/asset-service/src/generateAssetData/optimism/index.ts

```diff
@@ -1,10 +1,19 @@
 import { optimismChainId } from '@shapeshiftoss/caip'
 
+import { getRenderedIdenticonBase64 } from '../../service/GenerateAssetIcon'
 import { optimism } from '../baseAssets'
 import * as coingecko from '../coingecko'
 
 export const getAssets = async () => {
   const assets = await coingecko.getAssets(optimismChainId)
   assets.push(optimism)
-  return assets
+  return assets.map((asset) => {
```

Comment:
And this guy.

Reply:
https://github.com/shapeshift/lib/pull/1216/commits/405ed66ca234589240d65344fd4d05d0eeed6994